### ISSUE TITLE
Create a sample audio captcha

### ISSUE THREAD

**Issue by hrj** [librecaptcha/lc-core#74]
We could use a Text-to-speech (TTS) system to generate an audio sample, and store it using a well known format, like Opus, etc.

The stretch goal would be to do it in such a way that voice recognition software can't recognise it. So we might have to add some distortions to the audio.

**Comment by myleskeeffe** [comment]
I don’t know if you *necessarily* need to add Voice-to-Text preventions - as a VTT translation would use a fair amount of machine power - so anything more then just a few bots would be way to slow and taxing on a spammer *id hope* hehe

**Comment by hrj** [comment]
@myleskeeffe Good point. Maybe just adding a bit of white noise to the clip might be more than enough, and push the cost for the bots, with very little cost on our end.

**Comment by myleskeeffe** [comment]
Sounds like a good idea - and depending which TTS library is used - could use multiple ‘accents’ to prevent bots just recognising how a word is said (I mean if we had a dictionary of like 30k words - it’d be pretty unlikely for someone to make a bot that matches their own library of the TTS recordings (as that’d grow to a fair few gbs) - and the bit of white noise would prevent them from just running a direct match against such a library anyways)

**Comment by hrj** [comment]
Hi

We are currently not actively working on the project due to constraints.

Are people interested in sponsoring work on this and other issues? If yes, I might be able to find folks or spend time on it myself.

React with a 👍🏽 if you or your organisation are willing to sponsor.

**Comment by oopsliu** [comment]
Thanks @hrj , I need to talk about this within my organization regarding sponsorship.
